app/util/ClassToMap.scala

```diff
@@ -0,0 +1,101 @@
+package util
+
+import com.gu.contentatom.thrift.atom.media._
+import com.gu.contentatom.thrift.{ImageAssetDimensions, ImageAsset, Image}
+import shapeless.labelled._
+import shapeless._
+
+object ClassToMap {
+  // FROM CASE CLASS TO MAP (http://stackoverflow.com/a/31638390)
+  trait ToMapRec[L <: HList] {
+    def apply(l: L): Map[String, Any]
+  }
+
+  // Case for heads that don't have ToMapRec instances (hconsToMapRec1).
```

Comment:
The heads that do have ToMapRec instances get these from the shapeless library right?